diff --git a/.gitattributes b/.gitattributes
index c9d292d..68f2020 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -1,3 +1,7 @@
+# Remal GitHub actions:
+dist/** -diff linguist-generated=true
+
+
 # Common:
 * text=auto eol=lf
 
diff --git a/.gitignore b/.gitignore
index 4d145d7..4f091ba 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,8 @@
+# Remal GitHub actions:
+/build
+/coverage
+
+
 # Logs:
 log/
 !**/src/**/log/
